requirements.txt

```diff
@@ -13,7 +13,7 @@ scipy
 tqdm
 scikit-image
 scikit-gstat>=1.0
-geoutils==0.0.13
 pip
+geoutils
```

Comment:
Why is the version removed? This means installs might get weird since there's a minimum version requirement of 0.0.13 (?).

Reply:
This will revert back with `environment.yml` when removing the current `git+geoutils` for CI tests! 

Right now I just added an exception in the `pre-commit` for `conda_to_pip_deps` so that CI does run when pulling from Git, and that just adds this versionless `geoutils`.